### PR TITLE
More mods

### PR DESCRIPTION
**Pull request by shedaniel**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,10 +19,145 @@ These are all the tools necessary to create a mod with Fabric.
 
 ## Mods made with Fabric
 These are mods that have been created with or ported to Fabric.
-- [CC: Tweaked](https://github.com/SquidDev-CC/CC-Tweaked/tree/mc-1.14-fabric): Just another ComputerCraft fork
-- [Galacticraft: Rewoven](https://github.com/teamgalacticraft/Galacticraft-Rewoven): Galacticraft: Rewoven for the FabricMC modding framework
-- [Hwyla](https://github.com/TehNut/HWYLA): A fork of Waila maintaining it for new MC versions
-- [ModMenu](https://github.com/Prospector/ModMenu): A menu for, you guessed it, mods!
+- [AbusedLib](https://minecraft.curseforge.com/projects/abusedlib): A lib mod that i use in some of my mods
```
Comment (UpcraftLP) [inline readme.md]: you should really not just copy the CF description without any context

Comment (shedaniel) [reply readme.md]: _Well they are auto-generated and I mean you can't get any better descriptions for that mod right_

Comment (UpcraftLP) [reply readme.md]: >you can't get any better descriptions for that mod right

well you, an `I` in there doesn't make sense without context, for example

Comment (shedaniel) [reply readme.md]: Ah right changed

**Pull request merged**